projects/plugins/jetpack/modules/woocommerce-analytics/class-jetpack-woocommerce-analytics.php

```diff
@@ -31,6 +33,13 @@ class Jetpack_WooCommerce_Analytics {
 	 */
 	private static $analytics = false;
 
+	/**
+	 * Instance of the My account functions
+	 *
+	 * @var Static property to hold concrete analytics impl that does the work.
```

Comment:
Nitpick, but let's use the full word `implementation` here just to slightly increase readability.